### ISSUE TITLE
Errors hen using arrows to navigate the autocompletion menu 

### ISSUE THREAD

**Issue by 0cv** [chriswalz/bit#27]
For example with `bit checkout ...`
![image](https://user-images.githubusercontent.com/1253866/95712284-cff2f080-0c64-11eb-9df7-a9f14322eeff.png)

Then when the last selection is hit it looks like this and the selection is blocked:
![image](https://user-images.githubusercontent.com/1253866/95712116-7a1e4880-0c64-11eb-92bc-f7756fad92a3.png)

Under certain circumstances (when coming back and forth, not sure exactly how), then I get also this:
![image](https://user-images.githubusercontent.com/1253866/95712533-29f3b600-0c65-11eb-8f36-496c0915858d.png)

**Comment by chriswalz** [comment]
What OS/Terminal/Shell are you using?

**Comment by 0cv** [comment]
You're right, I didn't realize it could be related to my Terminal and/or Shell. I was using the Terminal of VSCode. I'm using also iTerm with ZSH in OSX and it works including the default Terminal.app. But it fails with the embedded terminal of Code => 

![image](https://user-images.githubusercontent.com/1253866/95713688-45f85700-0c67-11eb-8531-0e7df9db0323.png)

**Comment by zeeke** [comment]
Hi, I think I reproduced this issue also with a basic go-prompt program.

In both ways (running bit and with adoc program) I can hit the problem only if I hold DOWN arrow button.

I'm not sure it's what @0cv experienced, but the root cause is that vscode sends two DOWN characters at once in [prompt.go](https://github.com/c-bata/go-prompt/blob/master/prompt.go#L278).

Just discovered adding logs:

```
2022/10/08 18:59:20 b: 1b5b42    <--- good DOWN arrow 
2022/10/08 18:59:21 b: 1b5b421b5b42  <--- bad 2xDOWN arrow 
```

See https://github.com/c-bata/go-prompt/blob/master/input.go#L72

Going to raise an issue also to go-prompt repo.
